src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -136,6 +139,29 @@ public void getPatronAccountById(String id, boolean includeLoans,
     }
   }
 
+  private CompletableFuture<String> getCurrencyCode(Map<String, String> okapiHeaders, VertxOkapiHttpClient httpClient) {
+    String path = "/configurations/entries";
```

Comment:
This request introduces a new dependency on the configuration interface.

Please add this dependency to the module descriptor and the relevant permissions to each endpoint that makes this request.

Reply:
Ah, of course.  Will do.